### ISSUE TITLE
MappingException: Error 729: ConflictWithOtherMechanisms

### ISSUE THREAD

**Issue by danielgerlag** [lontivero/Open.NAT#70]
Hi,

I am unable to map any ports, I always get this error
```
MappingException: Error 729: ConflictWithOtherMechanisms
```

**Comment by lontivero** [comment]
That's not an Open.NAT error message, it is an error received from your router and for that reason you won't be able to use any upnp library. 

This is what the UPNP:WanIPConnection service specification says: 

> NAT port mapping rules can be created by other mechanisms besides UPnP IGD. Therefore, it is possible that port mappings done by independent mechanisms MAY overlap or conflict. It is left to vendors to determine suitable algorithm on how to resolve conflicting mappings. A new error code has been created **(729 PortMappingNotAllowed)** in order to allow an IGD to deny a request due to conflict with other mechanisms.

It seems that your router has another mechanism for mapping ports which is in conflict with UPnP IGD. You should log into your router and review its configuration in order to change whatever has to be changed.

**Comment by danielgerlag** [comment]
Not true, I have applications on my network using upnp. Do you have a list of routers that your library is compatible with?

**Comment by lontivero** [comment]
I don't have a list of routers. The library implements the [specifications](http://upnp.org/specs/gw/UPnP-gw-WANIPConnection-v2-Service.pdf) so, it should compatible with routers that implement them too.  

If you have other applications doing port forwarding with upnp the, the conflict could be in one specific port, and not with all of them. Anyway, Open.NAT is simply the messager. Your router complains about a conflict somewhere. 

I can take a look at this problem if you share the network.log file here.
